.github/workflows/back-linter-status-check.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will do a clean installation of Python dependencies, cache/restore them,
+# and test linting on the back-end code
+
+name: Back-end Lint
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  lint:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.11]
+
+    defaults:
+      run:
+        working-directory: src-back
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+        cache: 'pip'
+
+    - name: Install dependencies
+      run: |
+        pip install --upgrade pip
+        pip install -r requirements.txt
+
+    # - run: cd src-back
+    - name: Lint with Flake8
+      run: flake8 .
```

Comment:
does this write to file or only perform a check? If it overwrites, it should not ideally.

Reply:
It only perform a check and doesn't change the code.